systers_portal/meetup/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import url
+
+from meetup.views import MeetupLocationAboutView
+
+
+urlpatterns = [
+    url(r'^(?P<slug>\w+)/about/$',
+        MeetupLocationAboutView.as_view(), name='about_meetup_location'),
```

Comment:
Because total chars are 101 in one line that is why I didn't updated it. But I am wondering flake is not warning for 101 chars.